pvnet/utils.py

```diff
@@ -69,6 +69,34 @@ def __call__(self, gsp_id: int) -> Location:
         )
 
 
+class SiteLocationLookup:
```

Comment:
Not added unit tests for this as this is used only in a single place (site backtest script) so far and is similar to the class above for GSP location lookup (not quite similar enough though IMO that it's worth merging them into one class)

Reply:
would it be easy to add a quick test, just incase? Makes it a bit more reobust for the future

Reply:
I've now added a parametrised unit test for this